lxd/instance/drivers/driver_qemu.go

```diff
@@ -1371,34 +1372,21 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 					sessionDataFDOk = true
 				}
 
-				addSEVCmd := func(policy string) {
-					var optSessionFiles string
-					if dhCertFDOk && sessionDataFDOk {
-						optSessionFiles = fmt.Sprintf("dh-cert-file=/proc/fd/%d,session-file=/proc/fd/%d,", dhCertFD, sessionDataFD)
-					}
-
-					qemuCmd = append(qemuCmd, "-machine", "q35,confidential-guest-support=sev0")
-					qemuCmd = append(
-						qemuCmd,
-						"-object",
-						fmt.Sprintf(
-							"sev-guest,id=sev0,cbitpos=%d,reduced-phys-bits=%d,%spolicy=%s",
-							cbitpos,
-							reducedPhysBits,
-							optSessionFiles,
-							policy,
-						),
-					)
+				sevOpts.cbitpos = cbitpos
```

Comment:
This stuff should be merged into the original commit so we don't introduce logic that we remove two commits later in the same branch. This causes time wastage during review as we go from commit to commit and therefore leave unapplicable comments.